apps/meteor/server/models/raw/Rooms.ts

```diff
@@ -1707,7 +1707,31 @@ export class RoomsRaw extends BaseRaw<IRoom> implements IRoomsModel {
 		return this.updateOne(query, update);
 	}
 
-	unmuteUsernameByRoomId(_id: IRoom['_id'], username: IUser['username']): Promise<UpdateResult> {
+	muteReadOnlyMutedUsernameByRoomId(_id: IRoom['_id'], username: IUser['username']): Promise<UpdateResult> {
+		const query: Filter<IRoom> = { _id };
```

Comment:
Doesn't it make sense to query also by readonly here just to make sure and to make the method name make more sense? `const query: Filter<IRoom> = { _id, ro: true };`